src/controllers/handlers/comms-scene-handler.ts

```diff
@@ -46,6 +52,22 @@ export async function commsSceneHandler(
   const credentials = await livekit.generateCredentials(identity, room, permissions, forPreview)
   logger.debug(`Token generated for ${identity} to join room ${room}`)
 
+  const event: UserJoinedRoomEvent = {
+    type: Events.Type.COMMS,
+    subType: Events.SubType.Comms.USER_JOINED_ROOM,
+    key: `user-joined-room-${room}`,
+    timestamp: Date.now(),
+    metadata: {
+      sceneId: sceneId || '',
+      userAddress: identity.toLowerCase(),
+      parcel,
+      realmName,
+      isWorld
+    }
+  }
+
+  await publisher.publishMessages([event])
```

Comment:
What about:

```suggestion
  setImmediate(async () => {
  const event: UserJoinedRoomEvent = {
    type: Events.Type.COMMS,
    subType: Events.SubType.Comms.USER_JOINED_ROOM,
    key: `user-joined-room-${room}`,
    timestamp: Date.now(),
    metadata: {
      sceneId: sceneId || '',
      userAddress: identity.toLowerCase(),
      parcel,
      realmName,
      isWorld
    }
  }

  await publisher.publishMessages([event])
  })
```

To prevent waiting for this publish on the endpoint call